aiida/cmdline/commands/cmd_profile.py

```diff
@@ -92,22 +95,36 @@ def profile_setdefault(profile):
     help='Include deletion of associated file repository.'
 )
 @arguments.PROFILES(required=True)
-def profile_delete(force, include_config, include_db, include_repository, profiles):
-    """
-    Delete one or more profiles.
+@click.pass_context
+def profile_delete(ctx, force, include_config, include_database, include_repository, profiles):
+    """Delete one or more profiles.
 
-    You can specify more profile names (separated by spaces).
-    These will be removed from the aiida config file,
-    and the associated databases and file repositories will also be removed.
+    The PROFILES argument takes one or multiple profile names that will be deleted. Deletion here means that the profile
+    will be removed including its file repository and database and all their content. The various options can be used
+    to control which parts of the profile are deleted.
     """
-    from aiida.manage.configuration.setup import delete_profile
+    if not include_config:
+        echo.echo_deprecated('the `--skip-config` option is deprecated and is no longer respected.')
 
     for profile in profiles:
-        echo.echo_info(f"Deleting profile '{profile.name}'")
-        delete_profile(
-            profile,
-            non_interactive=force,
-            include_db=include_db,
-            include_repository=include_repository,
-            include_config=include_config
+
+        if not include_database and not include_repository:
+            deleting = '.'
```

Comment:
Since deleting the db and the repo is the default behavior (and rightfully so), I think you actually want to inform the user about things that are *not* being deleted.

Reply:
Done